scraper.rb

```diff
@@ -19,8 +19,9 @@ def scrape(h)
 end
 
 def scrape_list(term, url)
-  scrape(url => MembersPage).member_urls.each do |mem_url|
-    data = scrape(mem_url => MemberPage).to_h.merge(term: term)
+  scrape(url => MembersPage).seat_rows.each do |seat_row|
+    next if primary_member_url.empty?
```

Comment:
this is too much of a step in the wrong direction here. We live with these sorts of command methods even though it's not the preferred way we'd write them now, but when you're starting to tweak the logic of them like this, it's time to migrate them to return the data and then write it all out in a separate step. (NB: you might want to do that as a preparatory PR to this one…)